From 517be8d572d540b18440b7aabb2c3e489f25d9d3 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 25 Feb 2020 22:02:27 +0100
Subject: [PATCH v2 1/1] include/ata.h: remove invalid links

Remove invalid URLs for "Information Technology - AT Attachment-3 Interface
(ATA-3)", point to ANSI X3.298-1997.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2:
	refer to the ANSI standard
---
 include/ata.h | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/include/ata.h b/include/ata.h
index 3f4e4a0234..d8a0275630 100644
--- a/include/ata.h
+++ b/include/ata.h
@@ -6,11 +6,8 @@
 
 /*
  * Most of the following information was derived from the document
- * "Information Technology - AT Attachment-3 Interface (ATA-3)"
- * which can be found at:
- * http://www.dt.wdc.com/ata/ata-3/ata3r5v.zip
- * ftp://poctok.iae.nsk.su/pub/asm/Documents/IDE/ATA3R5V.ZIP
- * ftp://ftp.fee.vutbr.cz/pub/doc/io/ata/ata-3/ata3r5v.zip
+ * "Information Technology - AT Attachment-3 Interface (ATA-3)",
+ * ANSI X3.298-1997.
  */
 
 #ifndef	_ATA_H
-- 
2.25.0

